src/metaxy/metadata_store/duckdb.py

```diff
@@ -165,3 +230,44 @@ def _create_native_components(self):
         diff_resolver = NarwhalsDiffResolver()
 
         return joiner, calculator, diff_resolver
+
+    # ------------------------------------------------------------------ DuckLake
+    def open(self) -> None:
```

Comment:
probably needs a corresponding close. This has to be a `@contextmanager` for simplicity